io/net/iperf_test.py

```diff
@@ -157,10 +177,7 @@ def tearDown(self):
         except Exception:
             self.peer_public_networkinterface.set_mtu('1500')
         self.networkinterface.remove_ipaddr(self.ipaddr, self.netmask)
-        try:
-            self.networkinterface.restore_from_backup()
-        except Exception:
-            self.log.info("backup file not availbale, could not restore file.")
+        self.networkinterface.restore_from_backup()
```

Comment:
@vaishnavibhat this is also same, don't remove try ..except. we handle this as we are getting exception if backup file was not there.